code/modules/mob/living/simple_animal/hostile/statue.dm

```diff
@@ -222,6 +222,6 @@
 	faction -= "neutral"
 
 /mob/living/simple_animal/hostile/statue/restrained()
-	. == ..()
```

Comment:
No you did it properly don't worry
It's just that the mistake you fixed was funny